src/components/Editor/index.js

```diff
@@ -170,9 +181,22 @@ class Editor extends Component {
     this.selection = selectionRange(this.ref)
   }
 
+  onFocus = evt =>{
+    if(this.tabGuarded && document.activeElement === this.ref){
+      this.ref.classList.add("tabGuarded")
+    }
+    return evt
+  }
+  onBlur = evt => {
+    this.tabGuarded =true
+    this.ref.classList.remove("tabGuarded")
+    return evt
```

Comment:
This and the above doesn't need to return anything ;)

But we need to proxy both callbacks to potential props like the existing ones do:

```js
if (this.props.onKeyDown) {
  this.props.onKeyDown(evt)
}
```